core/src/main/scala/org/apache/spark/scheduler/cluster/mesos/MesosClusterScheduler.scala

```diff
@@ -422,6 +422,37 @@ private[spark] class MesosClusterScheduler(
     desc.schedulerProperties.get("spark.cores.max").map { v =>
       options ++= Seq("--total-executor-cores", v)
     }
+
+    /**
+      * The properties to pass along from Spark Submit to the Mesos Scheduler.
+      * http://spark.apache.org/docs/1.5.1/configuration.html#available-properties
+      */
+    val propertiesToPropagate = Set(
+      "spark.driver.extraClassPath",
```

Comment:
I agree with @tnachen that it's probably better to include all of them, and blacklist the ones that don't make sense. Either way it's a bit of a maintenance burden whenever a new property gets added, but it's more likely that we'd need it to be passed down than not.